packages/foreman/pcp-mmvstatsd/pcp-mmvstatsd.spec

```diff
@@ -34,6 +34,7 @@ ln -fs $(pwd) _gopath/src
 export GOPATH=$(pwd)/_gopath
 pushd _gopath/src/%{name}-%{version}
 go build
+strip %{name}
```

Comment:
Looks like this needs to be `%{name}-%{version}` according to the `install` command below